packages/ERTP/package.json

```diff
@@ -11,9 +11,9 @@
     "test": "ava",
     "pretty-fix": "prettier --write '**/*.js'",
     "pretty-check": "prettier --check '**/*.js'",
-    "lint-fix": "yarn lint --fix",
+    "lint-fix": "eslint --fix '**/*.js' && yarn lint:types",
     "lint-check": "yarn lint",
-    "lint": "yarn lint:types && eslint '**/*.js'",
+    "lint": "eslint '**/*.js' && yarn lint:types",
```

Comment:
Since you've changed `lint-fix` to behave the way you like, I would appreciate it if you left `lint` intact so that I can use `yarn lint --fix` as I intended.